src/GovUk.Education.ExploreEducationStatistics.Common/Services/BlobStorageService.cs

```diff
@@ -40,12 +42,22 @@ public class BlobStorageService : IBlobStorageService
         private readonly string _connectionString;
         private readonly BlobServiceClient _client;
         private readonly ILogger<IBlobStorageService> _logger;
+        private readonly IStorageInstanceCreationUtil _storageInstanceCreationUtil = new StorageInstanceCreationUtil();
 
-        public BlobStorageService(string connectionString, BlobServiceClient client, ILogger<IBlobStorageService> logger)
+        public BlobStorageService(
+            string connectionString,
+            BlobServiceClient client,
+            ILogger<IBlobStorageService> logger,
+            IStorageInstanceCreationUtil? storageInstanceCreationUtil = null)
         {
             _connectionString = connectionString;
             _client = client;
             _logger = logger;
+
+            if (storageInstanceCreationUtil != null)
```

Comment:
You could use the same syntax that we use in tests?

```
_storageInstanceCreationUtil = storageInstanceCreationUtil ?? new StorageInstanceCreationUtil()
```

Alternatively, you could set up DI to give you an instance of storageInstanceCreationUtil as per any other autowired dependency - it doesn't store any state (other than the static, so it's not unique to any instance of StorageInstanceCreationUtil, so it wouldn't matter if it's a singleton, a transient, or a new one for every wring event.